### PR TITLE
[Android] Fixed foreground FCM banner notifications and notification sound

### PR DESCRIPTION
**Pull request by tankers746**

### CODE REVIEW

**Comment by zabojad** [comment]
@tankers746 I tried your PR and while it fixes custom sounds on Android, it also has a side effect: local notification does not show up at the top of the screen anymore when triggered (they stack in the notification center quietly).

**Comment by zabojad** [comment]
@tankers746 OK, I have found why but it appears to be another existing bug that was leading to this side effect. The notification `importance` param value was not passed to the notification channel creation. I've fixed it on my fork there : https://github.com/zabojad/react-native-push-notification/commit/286e7d36fda47e44502980c2d266ee76a40d1ac5

This issue was showing up with you fix because your code creates a new channel when using a custom sound.

**Comment by TylerWardle** [comment]
@zabojad any chance you'd be putting in a PR of your fork?

**Comment by HarshitMadhav** [comment]
This commit is not merged to master.

**Comment by iamshadmirza** [comment]
Please merge this PR to master

**Comment by dabakovich** [comment]
@zo0r please check and merge the PR, it really works.

**Comment by Dallas62** [comment]
@tankers746 Thanks for your implication, can you summarize which bugs is fixed ?
As I understand:
- The Notification does appear in the notification center if the app is in Foreground
- Custom Sounds are now fixed at notification level on Android

@HarshitMadhav @iamshadmirza @dabakovich 
Did you test this PR?

Thanks!

**Comment by dabakovich** [comment]
@Dallas62 @tankers746 
Yes, tested it. The fix works for me. Hovewer I added an optional parameter `showInForeground` for Android platform so I am able now to choose whether to show this notification in foreground.
[https://github.com/dabakovich/react-native-push-notification/commit/06dcd7af2f164c38143a34f5eef14a5940c1b583](commit)

**Comment by Dallas62** [comment]
@dabakovich thank for this reply,
I saw this parameter in another PR if I remember well, but someone says that it should be handle in the JS side, using a `.postLocalNotification`.
If you do it in the JS side, I think the behavior is the same in iOS and Android. Has I remember, iOS does not show the notification if the App is in foreground, do you confirm?

**Comment by Dallas62** [comment]
Thanks for this PR!
I integrated it in master, but with some changes (missing description on channel and remove TMP variable).
